supervisor/os/manager.py

```diff
@@ -93,17 +96,17 @@ def to_dict(self) -> SlotStatusDataType:
         if self.sha256 is not None:
             out["sha256"] = self.sha256
         if self.size is not None:
-            out["size"] = self.size
+            out["size"] = cast(c_uint64, self.size)
```

Comment:
Are you sure it wouldn't be better to actually runtime convert this values here? If we don't convert them, does D-Bus next maybe do those later on or how does that work? If it anyway needs to be converted at runtime to c_uint64 later on, I think it would be better to do it here.

Reply:
This method isn't used. Removing it instead to simplify things